src/main/java/org/jabref/gui/exporter/ExportCustomizationDialogViewModel.java

```diff
@@ -0,0 +1,98 @@
+package org.jabref.gui.exporter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.gui.DialogService;
+import org.jabref.logic.exporter.TemplateExporter;
+import org.jabref.logic.journals.JournalAbbreviationLoader;
+import org.jabref.preferences.PreferencesService;
+
+public class ExportCustomizationDialogViewModel extends AbstractViewModel {
+
+    //TODO: Use JavaFX SortedList instead of SimpleListPreperty for exporters.  There is no SimpleSortedListProperty.
+    private final ListProperty<ExporterViewModel> exporters = new SimpleListProperty<>(FXCollections.observableArrayList());
```

Comment:
A sorted list is read only, you need to wrap it around an existing observable list